packages/inventory/src/services/inventory.ts

```diff
@@ -47,7 +47,7 @@ export default class InventoryService
     options?: unknown,
     moduleDeclaration?: ConfigurableModuleDeclaration
   ) {
-    super(arguments[0])
+    super(arguments[0], arguments[1], arguments[2])
```

Comment:
Suggestion(non blocking): either use ...arguments, or you can pass argument[0], options, configModule.

We use arguments[0] only as a shortcut to be sure to pass the entire container and not what as been destructured.